src/components/ContactDetails/ContactDetails.tsx

```diff
@@ -1,13 +1,126 @@
 import React from 'react';
 import { useParams } from 'react-router-dom';
+import {
+  makeStyles,
+  TextareaAutosize,
+  Typography,
+  CircularProgress,
+  Box,
+} from '@material-ui/core';
+import gql from 'graphql-tag';
+import { useQuery } from '@apollo/react-hooks';
+import NavBar from '../NavBar';
+import Avatar from '../Avatar';
+import { ReactComponent as EllipseShape } from '../../svg/ellipse-shape-gray.svg';
+import { ROUTES } from '../../shared/routes';
+import { ContactDetailsParams, ContactDetailsProps, NewbieData } from './types';
 
-const ContactDetails: React.FC = () => {
-  const { id: newbieId } = useParams();
+const useStyles = makeStyles(theme => ({
+  bold: {
+    fontWeight: 700,
+  },
+  notesTextarea: {
+    width: 150,
+    minHeight: 100,
+    borderWidth: '1px',
+    borderColor: theme.palette.primary.dark,
+  },
+  ellipse: {
+    position: 'fixed',
+    bottom: '-285px',
+    left: '-56px',
+  },
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginTop: '2rem',
```

Comment:
You can use here `theme.spacing(x)` function :)

Reply:
Done here: https://github.com/BuildIt-Poland/buddy-app/pull/21/commits/d7412a4124e56f72710eb43e6ff3f3286905d1d5 and here: https://github.com/BuildIt-Poland/buddy-app/pull/21/commits/d84311a66dd86508fad577396a9983ac43d8dd92